tests/misc.gr

```diff
@@ -102,3 +102,43 @@ Assert("image.delete", image.delete(img) == nil)
 Assert("image.size of deleted image", image.size(img) == nil)
 // extra delete is fine
 Assert("image.delete", image.delete(img) == nil)
+
+i := 4 
+log("before changes", i)
+i += 4
+Assert("i should be 8",i==8)
+i -= 5 
+Assert("i should be 3",i==3)
+i *= i
+Assert("i should be 9",i==9)
+i /= 9.
+Assert("i should be 1.",i==1.)
+i /= 2
+Assert("i should be .5",i==.5)
+i = 1
+log("before changes", i)
+i&=2
+Assert("i should be 0",i==0)
```

Comment:
test for |& also use say

5 (0b0101) &= with say 12 (0b1100) to get 4 instead of 0 as more likely to not work vs other ops

and need at least 1 test for |=